packages/gestalt/src/MobileModal.js

```diff
@@ -0,0 +1,211 @@
+// @flow strict
```

Comment:
Doesn't need to happen in this PR necessarily, but I'd put this in a `/modal` directory. I think we should only include exported components in the top-level directory

Reply:
As soon as you import from the upper directory, it throws an error. I've tried! But you can only do that with files that are consumed in the upper level

Reply:
That lint rule should only care about importing from a parent directory. Check out the organization I did for Masonry — putting this file into `/Modal/MobileModal.js` and then importing that into `/Modal.js` should be fine